tools/devtool

```diff
@@ -675,6 +680,20 @@ cmd_build() {
     # Run the cargo build process inside the container.
     # We don't need any special privileges for the build phase, so we run the
     # container as the current user/group.
+
+    # Build seccompiler
+    run_devctr \
+        --user "$(id -u):$(id -g)" \
+        --workdir "$CTR_FC_ROOT_DIR" \
+        ${extra_args} \
+        -- \
+        cargo build -p seccomp --bin seccompiler \
+            --target-dir "$CTR_CARGO_SECCOMPILER_TARGET_DIR" \
+            "${cargo_args[@]}"
+    ret=$?
```

Comment:
would it make sense to only build this as a separate target? smth like `devtool build seccompiler`.

Otherwise I am afraid we'll keep building it unnecessarily.

Reply:
Firecracker needs to build seccompiler anyway, it's run as part of its build script.
So only one of these build procedures will actually do the build. The next one will reuse the same build folder so will take 0.02s.

You think this is ok?

Reply:
Hmm, I must have misunderstood then, I was under the impression that the seccompiler is a separate stand-alone binary.